### ISSUE TITLE
Collection not working

### ISSUE THREAD

**Issue by LordAxerion** [micah686/VnManager#30]
Whenever I try to set up a collection (in my case Fate/stay night), after adding all the executables and clicking done, nothing happens when I press submit. There is also no error, but when I open the "manage executables" window again, the list is empty again.

**Comment by micah686** [comment]
It's been a while since I worked on the collection code, but without digging into it again:
After you add the list of games and press submit, it should bring you back to the main form. After you press submit on that form, it should add the entries automatically.

It also does clear out the entries when you back to the manage executables screen again.

**Comment by LordAxerion** [comment]
The last step is exactly what does not work for me. I can add the *.exe file and submit the manage executables form, but when clicking submit on the main form nothing happens. Maybe I've configured something in the wrong way, therefore see the following screenshot.
![image](https://user-images.githubusercontent.com/48788659/110497582-df347800-80f6-11eb-897e-81cdd270f78d.png)

**Comment by protoniogg** [comment]
Same problem. I tried with the Higurashi question arc exes,

**Comment by micah686** [comment]
Yeah, I did notice that as well. But, I am trying to resolve multiple issues, so that fix might not get in until later.
